modules/UI/util/JitsiPopover.js

```diff
@@ -76,7 +76,9 @@ var JitsiPopover = (function () {
      */
     JitsiPopover.prototype.createPopover = function () {
         $("body").append(this.template);
-        $(".jitsipopover > .jitsipopover-content").html(this.options.content);
+        let popoverElem = $(".jitsipopover > .jitsipopover-content");
+        popoverElem.html(this.options.content);
+        APP.translation.translateElement(popoverElem);
```

Comment:
Isn't it going to be better if we translate the content outside of JitsiPopover class because the content is generated somewhere else? WDYT?